app/res/layout/screen_login.xml

```diff
@@ -86,21 +86,19 @@
                                 android:textSize="@dimen/text_medium"
                                 android:visibility="gone"/>
 
-                            <EditText
+                            <AutoCompleteTextView

```

Comment:
It's easy to turn off/on, right? My guess is that based on what I've seen it's more likely to cause state-confusion than help, so let's turn it off for now and think about how we want to manage this input in the future.